models/gif_cond_exp_multisynapse.cpp

```diff
@@ -521,15 +523,6 @@ nest::gif_cond_exp_multisynapse::init_buffers_()
 void
 nest::gif_cond_exp_multisynapse::calibrate()
 {
-
```

Comment:
Could you explain your changes at this point in more detail? Do I understand correctly that you can remove this here because the state vectors and buffers are now correctly resized in `set` and `init_buffers_`?

Reply:
I don't understand what you want me to remove. Is it in the calibrate function ?
In my commit, I fixed the initialization of the state vectors and move them to the correct functions.
I have also fixed the spike buffers initialization which was not adapted to multiple synapses.

Reply:
No need to remove anything, I just wanted some more explanation. It is fine for me now.